docs/user/integration/sap-cloud-logging/README.md

```diff
@@ -0,0 +1,157 @@
+
+## Prerequisites
+
+- A SAP Cloud Logging service with OpenTelemetry enabled to ingest distributed traces
+- A secret named `cls` in `cls-integration` namespace, holding the credentials and endpoints for the instance
+
+## Ship Logs to Cloud Logging Service
+
+The telemetry module supports convenient shipment of application and access logs using LogPipeline custom resources (see [Kyma Telemetry Application Logs Documentation](./../../02-logs)). The setup distinguishes application logs and access logs which can be configured independently.
+To enable shipping logs to Cloud Logging Service follow the below procedure:
+
+1. Deploy the LogPipeline for application logs:
+    ```
+    cat <<EOF | kubectl apply -n cls-integration -f -
+    apiVersion: telemetry.kyma-project.io/v1alpha1
+    kind: LogPipeline
+    metadata:
+      name: cls-application-logs
+    spec:
+      input:
+        application:
+          containers:
+            exclude:
+              - istio-proxy
+      output:
+        http:
+          dedot: true
+          host:
+            valueFrom:
+              secretKeyRef:
+                name: cls
+                namespace: cls-integration
+                key: ingest-mtls-endpoint
+          tls:
+            cert:
+              valueFrom:
+                secretKeyRef:
+                  name: cls
+                  namespace: cls-integration
+                  key: ingest-mtls-cert
+            key:
+              valueFrom:
+                secretKeyRef:
+                  name: cls
+                  namespace: cls-integration
+                  key: ingest-mtls-key
+          uri: /customindex/kyma
+    ```
+1. Deploy the LogPipeline for Istio access logs and enable access logs in Kyma:
+    ```
+    cat <<EOF | kubectl apply -n cls-integration -f -
+    apiVersion: telemetry.kyma-project.io/v1alpha1
+    kind: LogPipeline
+    metadata:
+      name: cls-access-logs
+    spec:
+      input:
+        application:
+          containers:
+            include:
+              - istio-proxy
+      output:
+        http:
+          dedot: true
+          host:
+            valueFrom:
+              secretKeyRef:
+                name: cls
+                namespace: cls-integration
+                key: ingest-mtls-endpoint
+          tls:
+            cert:
+              valueFrom:
+                secretKeyRef:
+                  name: cls
+                  namespace: cls-integration
+                  key: ingest-mtls-cert
+            key:
+              valueFrom:
+                secretKeyRef:
+                  name: cls
+                  namespace: cls-integration
+                  key: ingest-mtls-key
+          uri: /customindex/istio-envoy-kyma
+    ```
+   Kyma sets Istio access logs to disabled by default. To enable Istio access logs selectively for your workload, follow the [access logs guide](https://kyma-project.io/docs/kyma/main/04-operation-guides/operations/obsv-03-enable-istio-access-logs/).
```

Comment:
Is "disabled" a value for the parameter? If, yes, use the code font as suggested. 

```suggestion
   Kyma sets Istio access logs to `disabled` by default. To enable Istio access logs selectively for your workload, follow the [access logs guide](https://kyma-project.io/docs/kyma/main/04-operation-guides/operations/obsv-03-enable-istio-access-logs/).
```

Reply:
no, it's not